pkg/i2gw/providers/ingressnginx/resource_reader.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/kubernetes-sigs/ingress2gateway/pkg/i2gw"
 )
 
-// converter implements the i2gw.CustomResourceReader interface.
+// converter implements the i2gw.CustomResourceFetcher interface.
 type resourceReader struct {
```

Comment:
```suggestion
type resourceFetcher struct {
```

Reply:
We should rename all the `CustomResourceReader`s to `CustomResourceFetcher`. The same goes for Kong and Nginx and for all the `resource_reader.go` files (should be `resource_fetcher.go`)

Reply:
Yep, sorry for not catching this first time. Now I've renamed all references I believe.